CONTRIBUTING.rst

```diff
@@ -0,0 +1,105 @@
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/ciscoucs/ucsc_apis/issues.
+
+If you are reporting a bug, please include:
+
+* Console logs and stack trace, if any.
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Submitting pull requests to change the documentation or code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Changes can be proposed by sending a pull request (PR). A maintainer will
+review the changes and provide feedback.
+
+The pull request will be merged into the master branch after discussion.
+
+Please make sure to run the tests and that the tests pass before submitting the
+PR. Please keep in mind that some changes might not be merged if the
+maintainers decide they can't be merged.
+
+Please squash your commits to one commit per fix or feature. The resulting
+commit should have a single meaningful message.
+
+Additions/Modifications done to the apis layer should conform to the following:-
+- add/create/set/modify functions should return the corresponding Managed Object
+- as much as possible every api should ideally be accompanied with it's testcase
+ 
+Testing your code
+~~~~~~~~~~~~~~~~~
+Some test cases are written to be run against live hardware.  You will need a
+real server available to run them.  Edit the tests/connection/connection.cfg
+file and set the hostname, username, and password to match your server.  e.g.
+
+    [ucsccentral]
```

Comment:
In connection.cfg file we are writing ucscentral as section header for specifying the host details. So this is fine.